tests/Api/toggleLine.test.ts

```diff
@@ -0,0 +1,16 @@
+import type { App } from 'obsidian';
+import { tasksApiV1 } from '../../src/Api';
+import { toggleLine } from '../../src/Commands/ToggleDone';
+
+jest.mock('obsidian', () => ({
+    Modal: class Mock {},
+}));
+
+const app = {} as App;
+describe('APIv1 - toggleLine', () => {
+    const api = tasksApiV1(app);
+
+    it('should be the toggleLine function', () => {
+        expect(api.toggleLine).toBe(toggleLine);
```

Comment:
Could you please add a call to the API and check the behaviour. No need to test all combinations but just a simple smoke test to make sure that the API is doing something =)